openslides/locale/tinymce/cs.js

```diff
@@ -0,0 +1,219 @@
+tinymce.addI18n('cs',{
```

Comment:
Is this a copied file? If yes we should use bower and gulp for this so we do not have to care about updates.